compiler/output.bal

```diff
@@ -8,6 +8,7 @@ const OUTPUT_EXTENSION = ".ll";
 
 type OutputOptions record {
     string? target = ();
+    string? outDir = ();
```

Comment:
This does not belong here since it is not used by outputModule.

Reply:
fixed